content/03-reference/01-tools-and-interfaces/07-preview-features/001-mssql/010-mssql-connection-string.mdx

```diff
@@ -0,0 +1,52 @@
+---
+title: 'MSSQL connection string'
+metaTitle: 'MSSQL connection string (Preview)'
+metaDescription: ''
+---
+
+<TopBlock>
+
+The MSSQL connection string follows the [JDBC standard](https://docs.microsoft.com/en-us/sql/connect/jdbc/building-the-connection-url?view=sql-server-ver15)
+
+The following example uses integrated security (Windows):
+
+```
+sqlserver://HOST:PORT;database=DATABASE;integratedsecurity=true
+```
+
+The following example uses SQL authentication (username and password):
+
+```
+sqlserver://HOST:PORT;database=DATABASE;user=USER;password=PASSWORD
+```
+
+> **Note**: If you are connectiong from MacOS, add `encrypt=DANGER_PLAINTEXT` to disable TLS to ease 
+
+</TopBlock>
+
+
+## Arguments
+
+TODO
+
+| Argument name      | Required | Default                | Description                                                                                                        |
+| :----------------- | :------- | ---------------------- | ------------------------------------------------------------------------------------------------------------------ |
+| `database`<br />`initial catalog` | No       | `master`     | The database to connect to.  |
+| `username`<br />`user`<br />`uid`<br />`userid`  | No, if you use integrated security       |                     | Maximum number of seconds to wait for a new connection                                                             |
+| `password`<br />`pwd`   | No, if you use integrated security       |                     | Maximum number of seconds to wait until a single query terminates                                                  |
+| `encrypt`          | No       | `prefer`               | Configures whether to use TLS, possible values: `prefer`, `disable`, `require`                                     |
+| `Integrated Security`          | No       |                        | Path to the server certificate                                                                                     |
```

Comment:
`IntegratedSecurity`, enables usage of Windows login credentials instead of SQL Server. Works only on Windows.